File: CsharpServer/Trader.cs
namespace CsharpServer
{
    class Trader {
        bool hasStock;

        public Trader(int traderId) {
            TraderId = traderId;
        }

        public int TraderId {
            get;
        }

        public void SetHasStockToTrue() {
            hasStock = true;
        }

        public void SetHasStockToFalse() {
            hasStock = false;
        }

        public bool GetHasStock() {
            return hasStock;
        }
    }
}

File: JavaStockMarketServer/src/Trader.java
public class Trader {
    private final int traderId;
    private boolean hasStock;

    public Trader(int traderId) {
        this.traderId = traderId;
    }

    public int getTraderId() {
        return traderId;
    }

    public void setHasStockToTrue() {
        hasStock = true;
    }

    public void setHasStockToFalse() {
        hasStock = false;
    }

    public boolean getHasStock() {
        return hasStock;
    }
}
